package.json

```diff
@@ -42,10 +42,10 @@
     "@babel/preset-env": "7.25.0",
     "@babel/preset-react": "7.24.7",
     "debug": "^2.2.0",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
-    "react-redux": "^5.0.6",
-    "redux": "^4.2.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-redux": "9.1.2",
+    "redux": "^5.0.0",
```

Comment:
Can we update these versions to their latest (sensible) versions.
React and React-dom to 18.3.1 and react-redux to 9.2.0.

Would we be able to upgrade Redux to a newer version? The version used is over 6 years old.
What about React, can we (and should we) use 19?